vispy/scene/visuals.py

```diff
@@ -3,7 +3,9 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 """
 The classes in scene.visuals are visuals that may be added to a scenegraph
-using the methods and properties defined by vispy.scene.Node. 
+using the methods and properties defined in vispy.scene.Node such as name,
+visible, parent, children, etc...
```

Comment:
Yeah, it would. We can tackle it later. We should at least be able to do it for the `Scene` versions of visuals because we manually construct their docstrings anyway.